policy/hub.go

```diff
@@ -287,9 +287,11 @@ func (s *LocalServices) ComputeBundle(ctx context.Context, mpolicyObj *Policy) (
 		Props:    map[string]*explorer.Property{},
 	}
 
-	// we need to re-compute the asset filters
-	mpolicyObj.Filters = &explorer.Filters{
-		Items: map[string]*explorer.Mquery{},
+	if mpolicyObj.Filters == nil {
```

Comment:
I'm curious why we check if filters are set here? 

ComputeBundle's original job was to (amongst other thigns) re-compute the policy filters from its PolicyGroups, so technically it should always clear out this structure first.